deploy/utils.sh

```diff
@@ -144,7 +141,7 @@ configure_cli_pod() {
   $cli_with_timeout exec $conjur_cli_pod -- conjur authn login -u admin -p $CONJUR_ADMIN_PASSWORD
 }
 
-function deploy_env {
+configure_env_url() {
```

Comment:
what is `env_url`? seems that `conjur_url` (or something like that) is a better naming